integration/testnet/network.go

```diff
@@ -174,9 +176,39 @@ func (net *FlowNetwork) Result() *flow.ExecutionResult {
 // Start starts the network.
 func (net *FlowNetwork) Start(ctx context.Context) {
 	// makes it easier to see logs for a specific test case
-	net.t.Logf("%v starting flow network %v with docker containers for test case [%v]",
-		time.Now().UTC(), net.config.Name, net.t.Name())
+	net.log.Info().Msgf("starting flow network %v with docker containers",
+		net.config.Name)
+
+	// print all existing containers before starting our containers, Useful to debug the issue
+	// that the tests fail due to "port is already allocated"
+
+	cli, err := dockerclient.NewClientWithOpts(dockerclient.FromEnv, dockerclient.WithAPIVersionNegotiation())
+	if err != nil {
+		panic(err)
+	}
+
+	containers, err := cli.ContainerList(ctx, types.ContainerListOptions{})
+	if err != nil {
+		panic(err)
+	}
+
+	t := net.t
+	t.Logf("%v (%v) before starting flow network, found %v docker container", time.Now().UTC(), t.Name(), len(containers))
+
+	for _, container := range containers {
+		t.Logf("%v (%v) before starting flow network, found docker container: %v %v", time.Now().UTC(), t.Name(), container.Names, container.Ports)
```

Comment:
I think we generally use `:` to indicate the start of wrapped error message. I personally prefer to avoid other usages 
```suggestion
		t.Logf("%v (%v) before starting flow network, found docker container %v with ports %v", time.Now().UTC(), t.Name(), container.Names, container.Ports)
```